ci-automation/vendor-testing/qemu_update.sh

```diff
@@ -12,7 +12,7 @@ set -euo pipefail
 
 source ci-automation/vendor_test.sh
 
-if [ "$*" != "" ] && [ "$*" != "*" ] && [ "$*" != "cl.update.payload" ]; then
+if [ "$*" != "" ] && [ "$*" != "*" ] && [[ "$*" != *"cl.update.payload" ]]; then
```

Comment:
(This is not perfect, if both tests are rerun, it will only look at the second test's name but anyway it's just to skip the 20 retries if someone uses it wrong)

Reply:
Yeah, it's good enough. But maybe you could add what you wrote as a comment for this line?